item-spec/common-metadata.md

```diff
@@ -212,3 +213,29 @@ optical and short-wave IR bands are all 30 meters, but the panchromatic band is
 PlanetScope Ortho Tile Product has an `gsd` of 3.7 (or 4 if rounding), even though the pixel size of the images is 3.125.
 For example, one might choose for WorldView-2 the Multispectral 20° off-nadir value of 2.07
 and for WorldView-3 the Multispectral 20° off-nadir value of 1.38.
+
+### Link Object
+
+This object describes a relationship with another entity. Data providers are advised to be liberal
+with the links section, to describe things like the Catalog an Item is in, related Items, parent or
+child Items (modeled in different ways, like an 'acquisition' or derived data).
+It is allowed to add additional fields such as a `title` and `type`.
```

Comment:
Is this sentence correct? `title` and `type` are both fields described in the table below.